frontend/test/lit/test_if_else.py

```diff
@@ -62,7 +62,7 @@ def otherwise():
 def test_convert_element_type(i: int, f: float):
     """Test the presence of convert_element_type JAX primitive when the type conversion is required"""
```

Comment:
```suggestion
    """Test the presence of convert_element_type JAX primitive
       when the type conversion is required"""
```

Reply:
fixed in https://github.com/PennyLaneAI/catalyst/pull/279/commits/fb3549bd2b7e17280a4575099df940e906d1c4d3